### PR TITLE
PR for List

### PR DESCRIPTION
**Pull request by suryxks**
features:

- ordered list styles
- unordered list styles
- stacked list

### CODE REVIEW

File: components/list/list.css [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+}
+.circle-list{
+    list-style: circle;
+}
+.unordered-list{
+    background-color: var(--white);
+    min-width: 15rem;
+    display: flex;
+    padding: 1rem;
+    margin: 0.5rem;
```
Comment (msdeshmukh009) [inline components/list/list.css:56]: You are repeating these styles so maybe create a single genric class like `.styled-list` and use it in both ordered and unordered lists.

**Review by msdeshmukh009 — APPROVED** [review]
The rest is looking good!!

**Review by someshtiwarii — APPROVED** [review]
looks good to go

File: components/list/list.css (hunk 1 above)
Comment (suryxks) [reply components/list/list.css:56]: Thank you mahesh for pointing it out have made the changes and will merge it

**Pull request merged**